### PR TITLE
OSDOCS:4654-Hide Backup Content

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-4654](https://issues.redhat.com//browse/OSDOCS-4654): Hide Backup Content

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4654

Link to docs preview:
file: https://55183--docspreview.netlify.app/https://55183--docspreview.netlify.app/
ROSA: https://55183--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.html 

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by jneczypor** [comment]
@yuwang-RH  Hey Yu and happy lunar new year! This PR relates to ROSA work, do you think you could take a look at confirm the changes? Thank you so much! Hope you had a really nice holiday and a break from work.

**Comment by jneczypor** [comment]
Hi @xueli181114! Aaren approved this on JIRA. Can you please add the QE approval when you have a free moment? Thanks so much.

**Review by bergerhoffer — COMMENTED** [review]
The actual updates look okay to me, but this seems like it's changing the support status of something in the already published docs. This might need to go through change management - did you happen to look into that? If not, @kalexand-rh can help determine if it's necessary

**Comment by kalexand-rh** [comment]
This looks like a large/important enough change to send through CM.

**Comment by jneczypor** [comment]
Hello @davemulford! Can I please get PX approval for this PR that needs to go through CM? Thanks so much!

**Comment by davemulford** [comment]
@jneczypor does this mean Red Hat will no longer back up the things we previously backed up, according to the Component table that was removed?

Also, what happens with any backups that were already performed for a non-STS cluster? Is Red Hat deleting that data?

**Comment by davemulford** [comment]
Okay thanks for that clarification! I was confused because the old docs had the following sections:
- Backup and recovery for ROSA clusters with STS
- Backup and recovery

The second item implying it was for non-STS clusters.

It looks like we collapsed data from these two sections into a single section named _Infrastructure and data resiliency_ and removed any mention of any backups. With that cleared up, LGTM.

**Comment by billmvt** [comment]
Confirming:

- ROSA & OSD still have Velero backups for all non-STS, non-FedRAMP clusters
- STS and FedRAMP clusters have never had Velero backups

**Comment by jneczypor** [comment]
Yes, as far as I know, the non-STS backup remains the same. And STS/FedRAMP backup also remains the same, we are just changing the documentation (that incorrectly showed STS backups that never existed).

**Comment by laujohns89** [comment]
lgtm

**Review by kelbrown20 — COMMENTED** [review]
Merge review LGTM! Awaiting last change management confirmations

File: modules/rosa-policy-incident.adoc [hunk 1]
```diff
@@ -41,49 +41,46 @@ The following activities can trigger notifications:
 - Upgrade scheduling
 
 [id="rosa-policy-backup-recovery-sts_{context}"]
-== Backup and recovery for ROSA clusters with STS
+== Infrastructure and data resiliency
+Customers are responsible for taking regular backups of their data and should deploy multi-AZ clusters with workloads that follow Kubernetes best practices to ensure high availability within a region. If an entire cloud region is unavailable, customers must install a new cluster in a different region and restore their apps using their backup data.
 
-There is no backup method available for ROSA clusters with STS.
+There is no backup method available for ROSA clusters with STS. Red Hat does not commit to any Recovery Point Objective (RPO) or Recovery Time Objective (RTO).
```
Comment (jewzaam) [inline modules/rosa-policy-incident.adoc:47]:
```suggestion
There is no Red Hat provided backup method available for ROSA clusters with STS. Red Hat does not commit to any Recovery Point Objective (RPO) or Recovery Time Objective (RTO).
```

Could like to https://docs.openshift.com/rosa/rosa_backing_up_and_restoring_applications/backing-up-applications.html#oadp-installing-oadp-rosa-sts_rosa-backing-up-applications ?

**Comment by jneczypor** [comment]
@kelbrown20 Hi Kelly! It has been over two days since since the CM email, and I made the necessary changes/have all the approvals. I think this can now be merged! Thank you!

**Pull request merged**